### PR TITLE
Open observe / vector

### PR DESCRIPTION
**Pull request by goastler**

### CODE REVIEW

**Comment by goastler** [comment]
blocked by #1433

File: docker/docker-compose.provider.yml [hunk 1]
```diff
@@ -17,13 +17,21 @@ services:
             options:
                 max-size: '100m'
                 max-file: '1'
+        healthcheck:
+            test: ["CMD", "curl", "--fail", "localhost:9229/v1/prosopo/provider/status"]  # ping the status api
```
Comment (forgetso) [inline docker/docker-compose.provider.yml:21]: Is this going to submit more notifications in the slack monitoring channel?

File: docker/docker-compose.provider.yml [hunk 2]
```diff
@@ -142,6 +157,34 @@ services:
             options:
                 max-size: '100m'
                 max-file: '1'
+    vector:
```
Comment (forgetso) [inline docker/docker-compose.provider.yml:160]: We'll need to see how staging copes with an additional container in terms of RAM / CPU when this is pushed to staging.

File: docker/images/vector/vector.toml [hunk 3]
```diff
@@ -0,0 +1,188 @@
+# enable vector's api
```
Comment (forgetso) [inline docker/images/vector/vector.toml:1]: Well done for deciphering this format. Looks horrendous.

**Review by forgetso — APPROVED** [review]
I think we just need to trust you on this one George and hope that it runs when released into staging. The configs are over my head but the docker files look ok.

**Pull request merged**